release-notes/swan-lake-alpha3-release-note.md

```diff
@@ -140,14 +140,32 @@ xml:Text nameText = (name/*).text();
 io:println(nameText); // "DanBrown"
 ```
 
+##### Bug Fixes
+
+To view bug fixes, see the [Github milestone issues](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
 #### Runtime
 
+##### Bug Fixes
+
+To view bug fixes, see the [Github milestone issues](https://github.com/ballerina-platform/ballerina-lang/issues?q=is%3Aissue+is%3Aclosed+milestone%3A%22Ballerina+Swan+Lake+-+Alpha3%22+label%3AType%2FBug+label%3ATeam%2FCompilerFE).
+
 #### Standard Library
 
+##### Bug Fixes
+
+To view bug fixes, see the [Github milestone issues](https://github.com/ballerina-platform/ballerina-standard-library/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+milestone%3A%22Swan+Lake+Alpha3%22).
+
 #### Code to Cloud
 
+##### Bug Fixes
+
+To view bug fixes, see the Github issues on [C2C](https://github.com/ballerina-platform/module-ballerina-c2c/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76+), [Docker](https://github.com/ballerina-platform/module-ballerina-docker/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76), [AWS Lambda](https://github.com/ballerina-platform/module-ballerinax-aws.lambda/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76), and [Azure Functions](https://github.com/ballerina-platform/module-ballerinax-azure.functions/issues?q=is%3Aissue+is%3Aclosed+label%3AType%2FBug+project%3Aballerina-platform%2F76) repos.
```

Comment:
Can we introduce the milestone for issues we close for alpha3? Or IMO we'll have to remove this for this release. @anuruddhal

Reply:
@MaryamZi - Why can't we have the Project link for this release?

Reply:
That defeats the purpose of including the bug fixes link. We include the bug fixes link so that the users can easily see what were the issues we fixed for that particular release. Without a milestone, there's no easy way to check what were the issues fixed for a particular release.

Reply:
@anuruddhal - Can we have the milestones created for Alpha3?

Reply:
We have to create 4 milestones for the 4 repos. I will create and update.

Reply:
Fixed.